tests/e2e/manifest/setup.sh

```
@@ -43,18 +43,18 @@ echo "Using namespace: $NAMESPACE"
 apply_template() {
   local template_file="$1"
   local target_namespace="$2"
-  
+
   echo "Applying template: $(basename "$template_file")"
-  
+
   # Substitute all variables and apply
-  sed -e "s/\${NAMESPACE}/$target_namespace/g" "$template_file" | kubectl apply -n $target_namespace -f - 
+  sed -e "s/\${NAMESPACE}/$target_namespace/g" "$template_file" | kubectl apply -n $target_namespace -f -
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)